oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/extractor/ExtractorSpec.java

```diff
@@ -267,14 +267,8 @@ public void slowSql(@DelegatesTo(SlowSqlSpec.class) final Closure<?> cl) {
         DatabaseSlowStatementBuilder builder = new DatabaseSlowStatementBuilder(namingControl);
         builder.setLayer(Layer.nameOf(log.getLayer()));
 
-        long timeBucket = TimeBucket.getTimeBucket(log.getTimestamp(), DownSampling.Minute);
         builder.setServiceName(log.getService());
 
-        ServiceMeta serviceMeta = new ServiceMeta();
-        String serviceName = namingControl.formatServiceName(log.getService());
-        serviceMeta.setName(serviceName);
-        serviceMeta.setLayer(builder.getLayer());
-        serviceMeta.setTimeBucket(builder.getTimeBucket());
```

Comment:
At here, `builder.getTimeBucket()` is always not initialized. Or the time bucket of ServiceTraffic is always wrong. OAP just fixed this due to this service's traffic also generated through MAL, which provides the correct timestamp.

Reply:
I think  `builder.getTimeBucket()` should be initialized by MAL such as 
![image](https://user-images.githubusercontent.com/74546965/209957220-2a2c5728-63ba-465c-996d-e3acd15ce4f5.png)

Reply:
That is `log.getTimestamp()`. Not this timestamp.

Reply:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/5441976/209957799-b3438531-9f44-4e43-a025-f714d717e5da.png">

Check the code history, you build a timestamp from it, but you never use it. Ths easy fix could be one line

![image](https://user-images.githubusercontent.com/5441976/209957965-9cf43b50-6857-4ce1-a189-c0eba8837c0e.png)

I changed others because you did duplicate `naming control` and are unnecessary to set `entity id`.

Reply:
I found the problem and it was my mistake. Sorry about that.

Reply:
> I found the problem and it was my mistake. Sorry about that.

You don't need to be `sorry`. **Bugs** is a part of engineers' daily life. If SkyWalking needs one to say `sorry`, that would be me. I did the most damage to the project.
So, don't worry. I just want to double-check check I found the right bug, and fix it in a right way.

Reply:
I get it. Thank you!